__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const db = require("../db/connection");
+const request = require("supertest");
+const app = require("../app");
+const testData = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+describe("far-han-news tests", ()=> {
+    describe("/api/topics", () => {
+        test("GET 200: returns with a status of 200", () => {
+            return request(app).get("/api/topics").expect(200)
+        })
+        test("GET 200: should return with an array of topics", () => {
+            return request(app).get("/api/topics").expect(200).then(({body})=> {
+                const {topics} = body
+                expect(topics).toEqual(expect.any(Array))
+                topics.forEach((topic)=>{
+                    expect(topic).toHaveProperty("slug", expect.any(String))
+                    expect(topic).toHaveProperty("description", expect.any(String))
+                })
+            })
+        })
```

Comment:
When using a forEach make sure that you check the length of the array that is coming back before you iterate through, an empty array would cause this test to incorrectly pass.